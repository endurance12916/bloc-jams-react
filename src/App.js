import React, { Component } from 'react';
import Navbar from './components/navbar/navbar.js';

class App extends Component {
  constructor(props){
    super(props);
    this.state = {

    }
  }

  render() {
    return (
      <div>
        <Navbar />
        {this.props.children}
      </div>
    );
  }
}

export default App;