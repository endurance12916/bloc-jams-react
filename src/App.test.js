import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router'

it('renders without crashing', () => {
  const div = document.createElement('div');
    ReactDOM.render(<MemoryRouter><App /></MemoryRouter>, div);
});

it('renders a snapshot', () => {
    const wrapper = renderer.create(<MemoryRouter><App/></MemoryRouter>).toJSON();
    expect(wrapper).toMatchSnapshot();
});