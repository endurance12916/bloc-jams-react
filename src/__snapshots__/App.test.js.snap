exports[`test renders a snapshot 1`] = `
<div>
  <nav
    className="navbar">
    <a
      className="logo"
      href="/"
      onClick={[Function]}>
      <img
        alt="bloc jams logo"
        src="bloc_jams_logo.png" />
    </a>
    <div
      className="links-container">
      <a
        className="navbar-link"
        href="/collection"
        onClick={[Function]}>
        collection
      </a>
    </div>
  </nav>
</div>
`;
